packages/api/test/helpers/chai-plugins.ts

```diff
@@ -30,7 +30,7 @@ declare global {
 
 export default function chaiPlugins(_chai, utils) {
   utils.addProperty(chai.Assertion.prototype, 'customUserAgent', function () {
-    const userAgent = this._obj['user-agent'][0];
```

Comment:
`fetch-mock` doesn't return headers values as an array of strings, eg. `{ 'content-type': ['application/json'] } ` vs `{ 'content-type': 'application/json' } `